src/components/radio-button-group/__tests__/RadioButtonGroup-test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+import { RadioButtonGroup } from '../RadioButtonGroup';
```

Comment:
I'm wondering if we should also have the tests running against the built components ?